src/Jecoute/StatisticsProvider.php

```diff
@@ -36,7 +36,7 @@ private function createQuestions(Collection $questions): array
         /** @var SurveyQuestion $surveyQuestion */
         foreach ($questions as $surveyQuestion) {
             $question = $surveyQuestion->getQuestion();
-            $dataBySurveyQuestion = $this->dataAnswerRepository->findDataBySurveyQuestion($surveyQuestion);
+            $dataBySurveyQuestion = $this->questionRepository->findDataBySurveyQuestion($surveyQuestion);
```

Comment:
You can pass directly the `$question` here, and rename `findDataBySurveyQuestion` by `findDataQuestion`

Reply:
fixed